docs/.gitignore

```diff
@@ -14,4 +14,7 @@ __pycache__
 .vscode/
 .sphinx/styles/*
 .sphinx/vale.ini
-sp-docs/_build
\ No newline at end of file
+sp-docs/_build
+doc-cheat-sheet-myst.md
+doc-cheat-sheet.rst
+.sphinx/latex_elements_template.txt
```

Comment:
I think the `.gitignore` in the parent should cover these already?

Reply:
That's good to know, thanks! I ended up reverting the changes